source/administration/devops-command-center.rst

```diff
@@ -0,0 +1,284 @@
+=====================
+DevOps Command Center
+=====================
+
+
+.. contents:: Contents
+  :backlinks: top
+  :local:
+  :depth: 2
+
+Overview
+^^^^^^^
+
+Workplace messaging is often process-driven rather than completely ad-hoc, and these processes could be organizationally critical in crisis situations. Help
+teams collaborate to solve problems that are time-sensitive, recurring, situational.
+
+Incident Response is a Mattermost plugin designed to help organizations monitor, coordinate,
+and measure their incident response processes, increasing transparency, maximizing effectiveness, and ultimately saving costs by cutting down time taken to respond and resolve incidents.
+
+
+User's Guide
+^^^^^^^^^^^^^
+
+Incidents are events within Mattermost that are initiated to manage a specific situation/response in real-time. When the situation/response is complete, the incident is ended.
+Incident details are stored as logs, and events that occurred within the incident can be reused in future incidents as Playbooks and checklists.
+
+Starting incidents
+~~~~~~~~~~~~~~~~~~
+
+The incident channel is the central place for discussion related to the incident. You can start an incident in one of three ways:
+
+- Use the slash command */incident start* from any channel.
+- Use the message actions dropdown menu and select **Start incident**.
+- Use the context menu of a post and select **Start incident**.
+
+When you create an incident, the name provided is applied to the new incident channel that is created. You can also select an optional Playbook. The channel name can be used to
+search for the channel, and is also the name appended to the archived logs. If a Playbook is selected,
+the corresponding checklist is used as a template to start the incident. If no Playbook is selected, the incident starts with an empty checklist.
+
+The creator of an incident automatically becomes the incident Commander, responsible for managing the incident.
+
+The Commander of the incident can:
+
+- Select Playbooks
+- Create checklists
+- Invite people to the channel
+- Hand over the Commander role to another channel member
+
+The Commander of the channel must be a team/channel admin, and if the role is handed to another member they must also be a team/channel admin. To change Commanders,
```

Comment:
I'm not sure whether the commander of the channel (the current one or the new one) must be a team/channel admin or not.

Reply:
I've tested this and it doesn't look like that's the case so I've edited this, thank you!